e2e/packages/sync-test/data/callWithSignature.ts

```diff
@@ -2,51 +2,46 @@ import { Page } from "@playwright/test";
 import { GetContractReturnType, PublicClient, WalletClient } from "viem";
 import { AbiParametersToPrimitiveTypes, ExtractAbiFunction, ExtractAbiFunctionNames } from "abitype";
 
-const DelegationAbi = [
+const Abi = [
```

Comment:
any reason we can't import this from world-modules?

Reply:
I guess I didn't think to try it 🤯